### DISCUSSION TITLE
test run cases  and tutorials for ww3 6.07 and ww3 7.14

### DISCUSSION THREAD

**Discussion by JMSnyder1980** [NOAA-EMC/WW3#920] [General]
Do you have by any chance test run cases for ww3 6.07 and ww3 7.14?   Do you have any tutorials on how to run these?  It would be nice to have them to test if the executables are viable and also be able to learn more how to use these versions of the model by example.   Can you please provide?

**Comment by JessicaMeixner-NOAA** [comment]
There is a folder with regression tests in both the public release version 6.07 and in the current develop v7.14.  The [manual](https://github.com/NOAA-EMC/WW3/wiki/Manual) describes this in section 5.11
